cpp/src/hdbscan/detail/soft_clustering.cuh

```diff
@@ -396,7 +396,7 @@ void all_points_membership_vectors(const raft::handle_t& handle,
   size_t n = prediction_data.n_cols;
 
   if (batch_size > m) batch_size = m;
-  RAFT_EXPECTS(0 < batch_size && batch_size <= m, "Invalid batch_size. batch_size should be > 0 and <= the number of samples in the training data");
+  ASSERT(0 < batch_size && batch_size <= m, "Invalid batch_size. batch_size should be > 0 and <= the number of samples in the training data");
```

Comment:
Why the macro change?

Reply:
Because it is not a raft related exception. It is a condition that needs to be true within cuML itself.

Reply:
Which header is `ASSERT` being pulled from, do you know?

Reply:
It is from `raft/core/error.hpp`. But it looks like that will get replaced with `RAFT_EXPECTS`. Should I change the ASSERT back to `RAFT_EXPECTS`?

Reply:
Yes, that's what I expected